apis/apps/v1beta3/emqxenterprise_webhook.go

```diff
@@ -130,3 +146,17 @@ func (r *EmqxEnterprise) ValidateDelete() error {
 	// TODO(user): fill in your validation logic upon object deletion.
 	return nil
 }
+
+func validateUserNameAndPassword(new, old Emqx) error {
+	if new == nil || old == nil {
+		return nil
+	}
```

Comment:
The username and password will not be nil in the old resource, because set default value in func `Default()`

Reply:
Done